src/Options.js

```diff
@@ -322,6 +322,13 @@ export class Options {
         this.setOption(name, object[name]);
     });
     this.modules = object.modules || this.modules;
+    if (!!object.atscript) {
+      this.atscript = !!object.atscript;      
+    }
+    if (!!object.experimental) {
```

Comment:
Moving the 'this.atscript = !!object.atscript;' line to before the parsing of the rest of the object would solve this.  Is that approach acceptable?